### DISCUSSION TITLE
Training Booklet Spelling, Grammar & Punctuation

### DISCUSSION THREAD

**Discussion by jundapro** [MonashDeepNeuron/HPC-Training#40] [General]
## About

Thread for general spelling/grammar/punctuation issues in the HPC Training Book. 

## Format

This is the desired format for listing a spelling issue:

- [ ] Chapter X.X - Name
  - Sub-title/section: 
  - Line number in Markdown source:  <~ optional
  - Description:

The corresponding Markdown is:

```markdown
- [ ] Chapter X.X - Name
  - Sub-title/section: 
  - Line number in Markdown source:  <~ optional
  - Description:
```

> Note: The checkbox represents the whether the error has been corrected.

**Comment by jundapro** [comment]
- [x] Chapter 2.3 - Types & Variables
  - Subtitle: Bitwise Operators
  - Description: Spelling error in name of 'bitwise complement' operator `~`. 
"compliment" is not 'complement' :))

**Reply by oraqlle** [reply]
Thanks for bringing this up. I've edited the discussion to give a concise format so we can more uniform address this issue and any others people find.

**Reply by oraqlle** [reply]
- Resolved in #43

**Comment by oraqlle** [comment]
- [x] Chapter 2.3 - Types & Variables
  - Subtitle: Bitwise Operators
  - Line number in Markdown source: L:105-106
  - Description: Left and right shift operators are swapped incorrectly.

**Reply by oraqlle** [reply]
- Resolved in #43

**Comment by oraqlle** [comment]
- [x] Chapter 1.6 - Challenges
  - Subtitle: Setting Up Challenges Repository
  - Line number in Markdown source: L:21
  - Description: `<repo-name>` for `upstream` remote add should have the actual link to the HPC Challenges repository

**Reply by oraqlle** [reply]
- Resolved in #43

**Comment by oraqlle** [comment]
- [x] Chapter 1.6 - Challenges
  - Subtitle: Challenges Repository
  - Line number in Markdown source: L:63
  - Description: The insert marker 'branch-name' was missing its right angle bracket.

**Reply by oraqlle** [reply]
- Resolved in #43
